NDTensors/src/NDTensors.jl

```diff
@@ -220,13 +223,30 @@ function disable_tblis()
   return nothing
 end
 
+function backend_octavian()
+  throw(UndefVarError)
+end
+
 function __init__()
-  @require TBLIS = "48530278-0828-4a49-9772-0f3830dfa1e9" begin
-    enable_tblis()
-    include("tensoralgebra/tblis.jl")
-  end
-  @require Octavian = "6fd5a793-0b7e-452c-907f-f8bfe9c57db4" begin
-    include("linearalgebra/octavian.jl")
+  @static if !isdefined(Base, :get_extension)
+    @require TBLIS = "48530278-0828-4a49-9772-0f3830dfa1e9" begin
+      enable_tblis()
```

Comment:
Do we need this any more? It seems like `using TBLIS; using NDTensors` achieves the same thing. Maybe we want to deprecate it and remove it in a future release in case people are using it.